README.md

```diff
@@ -40,8 +40,8 @@ rows for example `Cursorfetchone()` or `Cursor.fetchmany()`. By default
 `trino.dbapi.Cursor.arraysize` accordingly.
 
 # Basic Authentication
-The `BasicAuthentication` class can be used to connect to a LDAP-configured Trino
-cluster:
+The `BasicAuthentication` class can be used to connect to Trino cluster which uses Password Authentication. Password authentication can be any of authentication method such as LDAP authentication, Password File Authentication and Salesforce Authentication. For details on password authentication please check [Authentication types](https://trino.io/docs/current/security/authentication-types.html) documentation
```

Comment:
@hashhar I will add more details for other auth methods but is this text good for `BasicAuthentication`? Any changes suggested here?